src/coreComponents/finiteElement/elementFormulations/LagrangeBasis1.hpp

```diff
@@ -422,13 +421,13 @@ class LagrangeBasis1
     {
       for( int a=0; a<2; ++a )
       {
-        N[ a*5 ]   = LagrangeBasis1::valueBubble( coords[0] ) *
-                     LagrangeBasis1::valueBubble( coords[1] ) *
-                     LagrangeBasis1::value( a, coords[2] );
+        N[ a*4+1 ]   = LagrangeBasis1::valueBubble( coords[0] ) *
```

Comment:
I think it would be better to avoid the for loop and explicitly write the six basis functions like

```c++
        N[ 0 ] = LagrangeBasis1::valueBubble( coords[0] ) *
                 LagrangeBasis1::value( 0, coords[1] ) *
                 LagrangeBasis1::valueBubble( coords[2] );
        N[ 1 ] = LagrangeBasis1::valueBubble( coords[0] ) *
                 LagrangeBasis1::valueBubble( coords[1] ) *
                 LagrangeBasis1::value( 0, coords[2] );
        N[ 2 ] = LagrangeBasis1::value( 0, coords[0] ) *
                 LagrangeBasis1::valueBubble( coords[1] ) *
                 LagrangeBasis1::valueBubble( coords[2] );
        N[ 3 ] = LagrangeBasis1::value( 1, coords[0] ) *
                 LagrangeBasis1::valueBubble( coords[1] ) *
                 LagrangeBasis1::valueBubble( coords[2] );
        N[ 4 ] = LagrangeBasis1::valueBubble( coords[0] ) *
                 LagrangeBasis1::value( 1, coords[1] ) *
                 LagrangeBasis1::valueBubble( coords[2] );
        N[ 5 ] = LagrangeBasis1::valueBubble( coords[0] ) *
                 LagrangeBasis1::valueBubble( coords[1] ) *
                 LagrangeBasis1::value( 1, coords[2] );
```